doc/v2/documentation_data_format.rst

```diff
@@ -782,6 +974,11 @@ Detailed field description
   exactly match the species names in the BNGL-generated network file, and no
   pattern matching will be performed.
 
+- ``name`` [STRING, OPTIONAL]
+
+  A human-readable name for the entity. This is optional and may be used for
+  reporting or visualization purposes.
```

Comment:
Say that if no name is provided it defaults to the petabEntityId?

Reply:
Fine to add, but it's anyways not clear where exactly it would be used.